README.md

```diff
@@ -1,10 +1,84 @@
-# Talks-presentations
-Catch up with the presentations! they will all be uploaded here.
+# DiveLane - Apple Pay for Ethereum
```

Comment:
You don'd has to change this official readme. it's not for only this project :)

please revert it